src/NuGet.Services.AzureSearch/SearchService/AzureSearchQuery.cs

```diff
@@ -0,0 +1,161 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace NuGet.Services.AzureSearch.SearchService
+{
+    /// <summary>
+    /// The type used to generate Azure Search Service queries. Used by <see cref="SearchTextBuilder"/>.
+    /// Given the query "fieldA:value1 value2":
+    /// 
+    ///   * "value1" is a field-scoped value 
```

Comment:
XML documentation has [markup](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/xmldoc/list) for lists.

Reply:
I'd rather avoid verbose XML syntax as we're not generating docs from these comments. Let me know if you feel strongly about this.